doc/changes/changes_0.0.1.md

```diff
@@ -7,3 +7,4 @@ This release adds the initial implementation of the secret store
 ## Changes
 
 * #1: Added secret store
+* #4: Used python toolbox
```

Comment:
```suggestion
* #4: Add python toolbox
```

Reply:
AFAIK we agreed to always write in past tense in changes file. 
Would it be OK for you to write "added python toolbox"?